src/views/ManageList.tsx

```diff
@@ -153,6 +199,24 @@ export function ManageList({ listPath }: Props) {
 						</button>
 					</form>
 					<Toaster />
+					<form onSubmit={(e) => handleInvite(e, listPath)}>
+						<label htmlFor="receipient-email">
+							<input
```

Comment:
suggestion: It would be nice to add the label value here. So that is shows it before the input.

```suggestion
<label htmlFor="recipient-email">
    Recipient Email:
    <input
```